### PR TITLE
List Azure AKS clusters endpoint

### LINKED ISSUE #8293: List AKS clusters
**User Story**
List AKS clusters

**Acceptance criteria**

### PR DESCRIPTION
**Pull request by imharshita**
Signed-off-by: Harshita <imharshita@github.com>

**What this PR does / why we need it**:
List Azure AKS clusters endpoint

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #8293 

**Does this PR introduce a user-facing change?**:

```release-note
GET /api/v2/providers/aks/clusters to list Azure AKS clusters
```

### CODE REVIEW

File: pkg/handler/v2/routes_v2.go [hunk 1]
```diff
@@ -64,6 +64,20 @@ func (r Routing) RegisterV2(mux *mux.Router, metrics common.ServerMetrics) {
...
 		Handler(r.getFeatureGates())
 
+	// Defines a set of HTTP endpoints for interacting with EKS clusters
+	mux.Methods(http.MethodGet).
+		Path("/providers/eks/clusters").
+		Handler(r.listEKSClusters())
+
+	mux.Methods(http.MethodGet).
+		Path("/providers/ec2/regions").
+		Handler(r.listEC2Regions())
```
Comment (zreigz) [inline pkg/handler/v2/routes_v2.go]: please move this endpoint to `	
// Defines a set of HTTP endpoint for interacting with
// various cloud providers`
 and change for `/providers/aws/regions`

**Review by zreigz — CHANGES_REQUESTED** [review]

File: pkg/handler/v2/routes_v2.go (hunk 1 above)
Comment (imharshita) [reply pkg/handler/v2/routes_v2.go]: done

**Review by kron4eg — COMMENTED** [review]
/approve

**Pull request merged**